src/components/com_kunena/template/crypsisb3/layouts/user/edit/profile/default.php

```diff
@@ -33,17 +37,12 @@
 					<?php echo JText::_('COM_KUNENA_MYPROFILE_BIRTHDATE'); ?>
 				</label>
 			</td>
-			<?php list ($birthYear, $birthMonth, $birthDay) = explode('-', $this->profile->birthdate); ?>
 			<td>
-				<span class="hasTip col-md-6 form-inline" title="<?php echo JText::_('COM_KUNENA_MYPROFILE_BIRTHDATE')
-					. '::' . JText::_('COM_KUNENA_MYPROFILE_BIRTHDATE_DESC'); ?>">
-					<input id="birthdate" class="form-control" type="text" size="4" maxlength="4"  name="birthdate1"
-					       value="<?php echo $this->escape($birthYear); ?>" />
-					<input class="form-control" type="text" size="2" maxlength="2" name="birthdate2"
-					       value="<?php echo $this->escape($birthMonth); ?>" />
-					<input class="form-control" type="text" size="2" maxlength="2" name="birthdate3"
-					       value="<?php echo $this->escape($birthDay); ?>" />
-				</span>
+				<div id="birthdate">
+        <div class="input-group date">
+          <input type="text" name="birthdate" data-date-format="mm/dd/yyyy" value="<?php echo $this->profile->birthdate == '0001-01-01' ? JFactory::getDate()->format('m/d/Y') : $this->profile->birthdate; ?>"> <span class="input-group-addon"><i class="glyphicon glyphicon-th"></i></span>
```

Comment:
class="input-group" is missing.
Maybe you can also add another default value then year 1.